src/kubernetes/server/server-deployment.yaml

```diff
@@ -17,7 +17,14 @@ spec:
       containers:
       - name: scdf-server
         image: springcloud/spring-cloud-dataflow-server:2.5.0.BUILD-SNAPSHOT
-        imagePullPolicy: Always
+        imagePullPolicy: IfNotPresent
+        volumeMounts:
+          - name: database
+            mountPath: /etc/secrets/database
+            readOnly: true
+          - name: harborsec
```

Comment:
given do not merge, assuming this and the below refs to "harborsec" wouldn't be included, yet a doc thing on how to add mounts